install.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/sh
+
+set -e
+
+temp_dir=$(mktemp -d /tmp/xh.XXXXXXXX)
+cd "$temp_dir"
+
+if [ "$(uname -s)" = "Darwin" ] && [ "$(uname -m)" = "x86_64" ]; then
+    target="x86_64-apple-darwin"
+elif [ "$(uname -s)" = "Linux" ] && [ "$(uname -m)" = "x86_64" ]; then
+    target="x86_64-unknown-linux-musl"
+elif [ "$(uname -s)" = "Linux" ] && [ "$(uname -m)" = "amd64" ]; then
+    target="x86_64-unknown-linux-musl"
+elif [ "$(uname -s)" = "Linux" ] && ( uname -m | grep -q "arm" ); then
```

Comment:
`uname -m` on my phone (in [Termux](https://termux.com/)) gives `aarch64`, so I don't think this covers it. What is it on a Raspberry Pi?

(There are also [ARM processors without AArch32 support](https://stackoverflow.com/questions/59379848/running-32-bit-arm-binary-on-aarch64-not-working-despite-config-compat) that wouldn't be able to run our binaries, but I don't think it's worth going out of our way to detect those.)

(Termux is a case where `/usr/local/bin` doesn't exist, but we check whether the directory exists either way so that's not a big deal.)

Reply:
>name -m on my phone (in Termux) gives aarch64, so I don't think this covers it. What is it on a Raspberry Pi?
On Raspberry Pi 4, it was `armv7l`.

`chezmoi`'s [install script](https://github.com/twpayne/chezmoi/blob/master/assets/scripts/install.sh#L122) matches `aarch64` and `armv*` to `arm` so maybe we should copy that?

Reply:
Debian has its own arch called `armhf`. I don't know if it shows up in `uname -m`.

Maybe this is best:
```suggestion
elif [ "$(uname -s)" = "Linux" ] && ( uname -m | grep -q -e '^arm' -e '^aarch' ); then
```